modules/rosa-creating-cluster.adoc

```diff
@@ -17,11 +17,13 @@ You have completed the installation prerequisites.
 
 Enter the following command to create your cluster with the default cluster settings.
```

Comment:
Also, this entire procedure module should have steps. This doesn't have a bullet/number at all. I'm assuming that any sentence that starts with "Enter..." is a step, so since there are 3 of those, then they should be numbered. (A single step procedure uses an unnumbered bullet).